parastell/parastell.py

```diff
@@ -291,6 +292,112 @@ def export_source_mesh(self, filename='source_mesh', export_dir=''):
             export_dir=export_dir
         )
 
+    def construct_nwl_build(
+        self, toroidal_extent, wall_s, mat_tag='nwl_geom', **kwargs
+    ):
+        """Construct special InVesselBuild class object for neutron wall loading
+        geometry.
+
+        Arguments:
+            toroidal_extent (float): angular extent to model in toroidal
+                direction [deg].
+            wall_s (float): closed flux surface label extrapolation at wall.
+            mat_tag (str): DAGMC material tag to use for neutron wall loading
+                geomery (defaults to 'nwl_geom').
+
+        Optional attributes:
+            repeat (int): number of times to repeat build segment for full model
+                (defaults to 0).
+            num_ribs (int): total number of ribs over which to loft for each
+                build segment (defaults to 61). Ribs are set at toroidal angles
+                interpolated between those specified in 'toroidal_angles' if
+                this value is greater than the number of entries in
+                'toroidal_angles'.
+            num_rib_pts (int): total number of points defining each rib spline
+                (defaults to 61). Points are set at poloidal angles interpolated
+                between those specified in 'poloidal_angles' if this value is
+                greater than the number of entries in 'poloidal_angles'.
+            scale (float): a scaling factor between the units of VMEC and [cm]
+                (defaults to m2cm = 100).
+        """
+        self.radial_build = ivb.RadialBuild(
+            np.linspace(0.0, toroidal_extent, num = 4),
+            np.linspace(0.0, 360.0, num = 4),
+            wall_s,
+            radial_build={},
+            logger=self._logger,
+            **kwargs
+        )
+        
+        if wall_s > 1.0:
+            self.radial_build.radial_build['nwl_geom'] = \
+                self.radial_build.radial_build.pop('sol')
```

Comment:
This `radial_build` is empty at this point.

Reply:
This conditional modifies `radial_build` to contain a `nwl_geom` component, getting rid of `plasma` and `sol` (if present), both of which get automatically added to `radial_build`.

An alternative workflow might be to have some flag for `RadialBuild` that indicates whether to add `plasma` and `sol`, defaulting to `True`. Then we could use the same `construct_invessel_build` method for both the in-vessel component and NWL geometries.

Reply:
If we implement the above workflow for `RadialBuild`, we wouldn't actually need to have explicit workflows for NWL geometry construction and export. Users could just construct and export a NWL geometry using the other tools.

For example, via the Python API:
```python
... IVB parameters...
radial_build = {
    'nwl_geom': np.zeros(len(toroidal_angles), len(poloidal_angles)),
    'mat_tag': Vacuum
}
nwl_geom = Stellarator(...)
nwl_geom.construct_invessel_build(
    ...
    radial_build,
    add_plasma_and_sol=False,
    ...
)
nwl_geom.export_invessel_build(export_cad_to_dagmc=False)
nwl_geom.export_dagmc(
    skip_imprint=True,
    filename='nwl_geom'
)
```
where the `add_plasma_and_sol` flag would default to True. Alternatively, via the command-line API
```
parastell config.yaml -i
```
where `config.yaml` has the appropriate geometry definition and flag shown for the Python API.

Reply:
> ... both of which get automatically added to `radial_build`.

Right! I forgot about the setter and how it automatically adds things!!!